### ISSUE TITLE
[web:multi-view] AccessibilityAnnouncements should be a singleton

### ISSUE THREAD

**Issue by yjbanov** [flutter/flutter#139272]
Announcements are audio coming out of the speakers or other devices capable of producing a notification to the user. An announcement is not tied to any one `FlutterView`. However, we are currently hanging `AccessibilityAnnouncements` off of the view object, which is wrong.

Code in question: https://github.com/flutter/engine/blob/fcb866befa76eb82a54c539a05503882724e8ee1/lib/web_ui/lib/src/engine/platform_dispatcher.dart#L637
